### PR TITLE
Ensure environment is passed through for kubectl

### LINKED ISSUE #255: unable to launch the editor "vi"
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:

When attempting to edit kubernetes resource using `kubectl` tarmak fails with following error:

```
error: unable to launch the editor "vi"
```

**What you expected to happen**:

I expected the `vi` or `vim` [or whatever other] `$EDITOR` to launch so I can edit the kubernetes resources

**How to reproduce it (as minimally and precisely as possible)**:

```
armak -c ./ kubectl edit clusterroles "psp:privileged"
DEBU[0000] trying to connect to https://127.0.0.1:57588  app=tarmak
DEBU[0000] connected to kubernetes api v1.8.8            app=tarmak
error: unable to launch the editor "vi"
```

**Anything else we need to know?**:

I reckon this is happening because the container which is launched when `kubectl` is used does not contain `vi` or `vim` or whatever `$EDITOR` in the container image hence kaboom.

**Environment**:
- Kubernetes version (use `kubectl version`): v1.8.8
- Cloud provider or hardware configuration**: AWS
- Install tools:  🛠
- Others: 👀

### PR DESCRIPTION
**Pull request by simonswine**
Fixes #255

```release-note
NONE
```

### CODE REVIEW

File: pkg/tarmak/kubectl/kubectl.go [hunk 1]
```diff
@@ -282,9 +282,10 @@ func (k *Kubectl) Kubectl(args []string) error {
 	}
 
 	cmd := exec.Command("kubectl", args...)
-	cmd.Env = []string{
+	cmd.Env = append(
+		os.Environ(),
 		fmt.Sprintf("KUBECONFIG=%s", k.ConfigPath()),
```
Comment (munnerz) [inline pkg/tarmak/kubectl/kubectl.go]: Just a thought - if the local user already has KUBECONFIG set to something non-empty, which value will take precedence?

Comment (simonswine) [reply pkg/tarmak/kubectl/kubectl.go]: From golang stdlib:

```
        // Env specifies the environment of the process.
        // Each entry is of the form "key=value".
        // If Env is nil, the new process uses the current process's
        // environment.
        // If Env contains duplicate environment keys, only the last
        // value in the slice for each duplicate key is used.
        Env []string
```

I think this behaviour changed with golang 1.9+

**Comment by simonswine** [comment]
@munnerz this PR was created after we updated the prow config :cry:

**Pull request merged**